src/olympia/users/tests/test_indexers.py

```diff
@@ -16,11 +16,11 @@ def _extract(self):
     def test_extract_attributes(self):
         self.user = UserProfile.objects.create(
             email='nobody@mozilla.org', username='nobody',
-            display_name=u'Nôbody', bio=u'My Bïo',
+            display_name=u'Nôbody', biography=u'My Bïo',
             homepage='http://example.com/', location=u'Nëverland',
             occupation='What')
         extracted = self._extract()
-        for attr in ('email', 'username', 'display_name', 'bio',
+        for attr in ('email', 'username', 'display_name', 'biography',
```

Comment:
mmm we'll need a reindex to pick up the new field correctly. Do we know if there is anything that searches through the users in ElasticSearch atm ? It will broke if the queries depend on the bio...